businesscentral-monitoring/image.yaml

```diff
@@ -11,7 +11,7 @@ envs:
     - name: "PRODUCT_VERSION"
       value: "7.0.0"
     - name: "BUSINESS_CENTRAL_MONITORING_DISTRIBUTION_ZIP"
-      value: "jboss-bpmsuite-7.0.0-business-central-monitoring-eap7.zip"
+      value: "jboss-bpmsuite-7.0.0.Beta01-business-central-monitoring-ee7.zip"
```

Comment:
What if we just used the name `jboss-bpmsuite-7.0.0-business-central-monitoring-eap7.zip` and only update the checksum? I know it makes it a bit less clear what actual version is being used, but it would make things easier as the file names would not change with every update. WDYT?

Reply:
That's perfectly OK from the build perspective. You can add also comments what is the source of the distribution, if it helps.

Reply:
Considering that I have uploaded the cache and it need to consistent with the PR, we can probably do it as it was
The naming difference is easy to identify in cacher as well.

Reply:
There is a nice trick with the cacher - it fetches artifacts by ID ignoring the name, so it does not need bo be consistent. Fetched artifact will be named as the specified "artifact key".

Reply:
Yeah, I would stick with just single zip name and e.g. add a comment next to the checksum saying what build is that. Sound good?

Reply:
I think it could cause more confusion by having the inconsistent name. We would also deliver the zip files with 7.0.0.Beta01 to the customer in the open shift package. 
So the accurate name will help a customer to understand the meaning of that filed.

Reply:
OK, I give up. I'll approve the PR.